Makefile

```diff
@@ -57,7 +57,12 @@ endif
 
 .DEFAULT: runc
 
-runc:
+runc: runc-dmz runc-bin
+
+runc-dmz:
+	$(CC) -o runc-dmz -static contrib/cmd/runc-dmz/runc-dmz.c
```

Comment:
If we are going to make this a hard dependency, it should be placed out of “contrib”